src/components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+
+function Pokemon({ endpoint }) {
+  const [ pokemonData, setPokemonData ] = useState({})
+
+  useEffect(() => {
+    async function fetchPokemon() {
+      try {
+        const result = await axios.get(endpoint);
+        setPokemonData(result.data);
+        console.log('result')
+      } catch (e) {
+        console.error(e);
+      }
+    }
+
+    if (endpoint) {
+      void fetchPokemon();
+    }
+
+  }, [endpoint])
+
+  return (
+    <>
+      {Object.keys(pokemonData).length > 0 && (
```

Comment:
Dit kan dan dus simpeler
```suggestion
      {pokemonData && (
```